arch/arm/src/stm32f7/stm32_bbsram.c

```diff
@@ -317,7 +317,7 @@ static int stm32_bbsram_open(FAR struct file *filep)
 static int stm32_bbsram_internal_close(FAR struct bbsramfh_s *bbf)
 {
   bbf->dirty = 0;
-  clock_gettime(CLOCK_REALTIME, &bbf->lastwrite);
+  clock_systime_timespec(&bbf->lastwrite);
```

Comment:
There is a code in board files
```
      syslog(LOG_EMERG, "There is a hard fault logged.\n");
      state = (desc.lastwrite.tv_sec || desc.lastwrite.tv_nsec) ?  OK : 1;

      syslog(LOG_INFO, "Fault Log info File No %d Length %d flags:0x%02x "
          "state:%d\n", (unsigned int)desc.fileno, (unsigned int) desc.len,
          (unsigned int)desc.flags, state);

      if (state == OK)
        {
          time_sec = desc.lastwrite.tv_sec + (desc.lastwrite.tv_nsec / 1e9);
          gmtime_r(&time_sec, &tt);
          strftime(buf, HEADER_TIME_FMT_LEN , HEADER_TIME_FMT , &tt);

          syslog(LOG_INFO, "Fault Logged on %s - Valid\n", buf);
        }
```
that reassembles string time stamp with `gmtime_r`. So in case if `CONFIG_CLOCK_TIMEKEEPING=y` the `clock_gettime` may be
```
#if defined(CONFIG_CLOCK_TIMEKEEPING)
      ret = clock_timekeeping_get_wall_time(tp);
#else
      ret = clock_systime_timespec(&ts);
```
So I think we should keep `clock_gettime(CLOCK_REALTIME, &bbf->lastwrite);` here and in similar places